packages/mobile/src/geth/geth.ts

```diff
@@ -120,9 +129,23 @@ async function initGeth() {
     }
 
     if (!(await ensureGenesisBlockWritten())) {
+      ;[meanMillisecs, trackCount] = trackMeanMillisecs(
+        initTime,
+        meanMillisecs,
+        trackCount,
+        TRACK_EVERY_GETH,
+        CustomEventNames.geth_failed_genesis_block
```

Comment:
For these kinds of events - I'm not sure we would expect this to happen very often, so I don't think we really need to call this function, right?

Reply:
Yep, this are more abnormal conditions.

Reply:
Okey, good idea, then only CustomEventNames.sync_complete